openedx/core/djangoapps/content/search/tests/test_handlers.py

```diff
@@ -134,7 +149,7 @@ def test_create_delete_library_block(self, meilisearch_client):
             "block_type": "problem",
             "context_key": "lib:orgA:lib_a",
             "org": "orgA",
-            "breadcrumbs": [{"display_name": "Library Org A"}],
+            "breadcrumbs": [{"display_name": "Library Org A", "usage_key": "lib:orgA:lib_a"}],
```

Comment:
This key `lib:org:lib_a` is a library key (a type of context key), not a usage key. So I think in this case (for library blocks), we should just leave it out. The root context is already in the document as `context_key`.

Reply:
Agreed and fixed: 7478009